internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -154,6 +155,26 @@ func (s *SchedulerManager) UpdateSchedulerConfig(ctx context.Context, scheduler
 	return isMajorUpdate, nil
 }
 
+func (s *SchedulerManager) CreateUpdateSchedulerOperation(ctx context.Context, scheduler *entities.Scheduler) (*operation.Operation, error) {
+	err := s.validateScheduler(scheduler)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = s.schedulerStorage.GetScheduler(ctx, scheduler.Name)
+	if err != nil {
+		return nil, fmt.Errorf("no scheduler found to be updated: %w", err)
+	}
+
+	scheduler.State = entities.StateCreating //TODO (guilhermelyra): what correct value should be set for state in this case?
```

Comment:
As we discussed before, I think we should get together and review the status and maybe create a machine state. Maybe a status like "updating" would make sense?

Reply:
@caiordjesus we have decided that state property from scheduler doesn't aggreegate value today. for now we are going to follow setting this value only with creating. Soon we'll see with this property will be removed or not.